src/backend/allocator/dmabuf.rs

```diff
@@ -0,0 +1,177 @@
+//! Module for [dmabuf](https://01.org/linuxgraphics/gfx-docs/drm/driver-api/dma-buf.html) buffers.
```

Comment:
Also, this would be nice to have an example (even if just a sentence explaining it) of why and when weak references to dmabufs are useful.